shell/platform/darwin/ios/framework/Source/platform_views_controller.mm

```diff
@@ -739,153 +873,203 @@ bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
-  // TODO(hellohuanlin): investigate if it is possible to skip unnecessary BringLayersIntoView.
-  if (![desired_platform_subviews isEqualToArray:existing_platform_subviews]) {
-    for (UIView* subview in desired_platform_subviews) {
+  // TODO(hellohuanlin): investigate if it is possible to skip unnecessary bringLayersIntoView.
+  if (![desiredPlatformSubviews isEqualToArray:existingPlatformSubviews]) {
+    for (UIView* subview in desiredPlatformSubviews) {
       // `addSubview` will automatically reorder subview if it is already added.
-      [flutter_view addSubview:subview];
+      [flutterView addSubview:subview];
     }
   }
 }
 
-std::shared_ptr<OverlayLayer> PlatformViewsController::GetExistingLayer() {
-  return layer_pool_->GetNextLayer();
+- (std::shared_ptr<flutter::OverlayLayer>)nextLayerInPool {
+  return self.layerPool->GetNextLayer();
 }
 
-void PlatformViewsController::CreateLayer(GrDirectContext* gr_context,
-                                          const std::shared_ptr<IOSContext>& ios_context,
-                                          MTLPixelFormat pixel_format) {
-  layer_pool_->CreateLayer(gr_context, ios_context, pixel_format);
+- (void)createLayerWithIosContext:(const std::shared_ptr<flutter::IOSContext>&)iosContext
+                        grContext:(GrDirectContext*)grContext
+                      pixelFormat:(MTLPixelFormat)pixelFormat {
+  self.layerPool->CreateLayer(grContext, iosContext, pixelFormat);
 }
 
-void PlatformViewsController::RemoveUnusedLayers(
-    const std::vector<std::shared_ptr<OverlayLayer>>& unused_layers,
-    const std::vector<int64_t>& composition_order) {
-  for (const std::shared_ptr<OverlayLayer>& layer : unused_layers) {
+- (void)removeUnusedLayers:(const std::vector<std::shared_ptr<flutter::OverlayLayer>>&)unusedLayers
+      withCompositionOrder:(const std::vector<int64_t>&)compositionOrder {
+  for (const std::shared_ptr<flutter::OverlayLayer>& layer : unusedLayers) {
     [layer->overlay_view_wrapper removeFromSuperview];
   }
 
-  std::unordered_set<int64_t> composition_order_set;
-  for (int64_t view_id : composition_order) {
-    composition_order_set.insert(view_id);
+  std::unordered_set<int64_t> compositionOrderSet;
+  for (int64_t viewId : compositionOrder) {
+    compositionOrderSet.insert(viewId);
   }
   // Remove unused platform views.
-  for (int64_t view_id : previous_composition_order_) {
-    if (composition_order_set.find(view_id) == composition_order_set.end()) {
-      UIView* platform_view_root = platform_views_[view_id].root_view;
-      [platform_view_root removeFromSuperview];
+  for (int64_t viewId : self.previousCompositionOrder) {
+    if (compositionOrderSet.find(viewId) == compositionOrderSet.end()) {
+      UIView* platformViewRoot = self.platformViews[viewId].root_view;
+      [platformViewRoot removeFromSuperview];
     }
   }
 }
 
-std::vector<UIView*> PlatformViewsController::GetViewsToDispose() {
+- (std::vector<UIView*>)uiViewsToDispose {
```

Comment:
This sounds like a getter, but it actually have side effect (e.g. it also updates other properties). Can we rename it to soemthing like `computeUIViewsToDispose` (and not to use dot syntax from caller side?)

Reply:
Done. Added detailed documentation as well. 

This method probably shouldn't even exist in its current form. The underlying data structures desperately need clean up/consolidation, and the method is doing too much. We can deal with it in a followup.